packages/shared/components/popups/send/SendConfirmationPopup.svelte

```diff
@@ -181,7 +180,7 @@
 
 <send-confirmation-popup class="w-full h-full space-y-6 flex flex-auto flex-col flex-shrink-0">
     <Text type={TextType.h3} fontWeight={FontWeight.semibold} classes="text-left"
-        >{localize(tokenSend ? 'popups.transaction.title' : 'popups.sendNft.confirmationTitle')}</Text
+        >{localize('popups.transaction.title')}</Text
```

Comment:
According to the linked task, the title should be "Send asset"

```suggestion
        >{localize('general.sendAsset')}</Text
```